.github/workflows/deploy.yml

```diff
@@ -0,0 +1,68 @@
+name: Deploy site to Pages
+
+on:
+  push:
+    branches: ["helixswap/main"]
+  pull_request:
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  # Build job
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "18"
+          registry-url: https://npm.pkg.github.com
+
+      - name: Install dependencies
+        run: |
+          yarn config set npmAuthToken ${{secrets.JAY_DEBUG}}
```

Comment:
One question is, I don't know why I can't install packages using `secrets.GITHUB_TOKEN` now.